terraform/modules/cloudfoundry/elb_apps.tf

```diff
@@ -63,6 +83,21 @@ resource "aws_lb_listener_certificate" "sites_pages_staging" {
   certificate_arn = var.sites_pages_staging_cert_id
 }
 
+resource "aws_lb_listener_certificate" "main_pages_dev" {
```

Comment:
maybe a big side-track from your current goals, but can we put all of this into a module so we don't have to think about it in every environment?

Reply:
I thought about it, but i don't have a good enough handle on the organization of this repo to do it and this will be the last environment so 🤷 . I'm happy either way.